src/containers/auth/EmailPasswordSignIn.jsx

```diff
@@ -1,50 +1,159 @@
 import React from "react";
-import { Form, Button, Col, Container } from "react-bootstrap";
+import { Form, Button, Col, Container, Modal } from "react-bootstrap";
 import { Redirect } from "react-router-dom";
 import { connect } from "react-redux";
-import { signIn } from "../../actions/authActions";
+import { signIn, resetPassword } from "../../actions/authActions";
 
 const EmailPasswordSignIn = (props) => {
   const [email, setEmail] = React.useState("");
   const [password, setPassword] = React.useState("");
+  const [confirmModalShow, setConfirmModalShow] = React.useState(false);
+  const [resetSuccessModalShow, setResetSuccessModalShow] = React.useState(
+    false
+  );
+
+  const validateEmail = (email) => {
+    var reEmail = /^(?:[\w\!\#\$\%\&\'\*\+\-\/\=\?\^\`\{\|\}\~]+\.)*[\w\!\#\$\%\&\'\*\+\-\/\=\?\^\`\{\|\}\~]+@(?:(?:(?:[a-zA-Z0-9](?:[a-zA-Z0-9\-](?!\.)){0,61}[a-zA-Z0-9]?\.)+[a-zA-Z0-9](?:[a-zA-Z0-9\-](?!$)){0,61}[a-zA-Z0-9]?)|(?:\[(?:(?:[01]?\d{1,2}|2[0-4]\d|25[0-5])\.){3}(?:[01]?\d{1,2}|2[0-4]\d|25[0-5])\]))$/;
```

Comment:
i don't think you need this, bootstrap email forms come with validation

give the demo a try at the very top of the page:
https://getbootstrap.com/docs/4.0/components/forms/

Reply:
ok, omitted it. can you please take a look again? @louisrli